### PR TITLE
Enable and fix mixed-comparison Decimal tests

### PR DESCRIPTION
**Pull request by GregoryTravis**
- Enable mixed comparison tests
- Switch from `Decimal_Utils.fromFloatExact` to `Decimal_Utils.fromFloat`
- Fix failing tests

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [ ] The documentation has been updated, if necessary.
- [ ] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [ ] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      [TypeScript](https://github.com/enso-org/enso/blob/develop/docs/style-guide/typescript.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- [ ] Unit tests have been written where possible.

### CODE REVIEW

File: distribution/lib/Standard/Table/0.0.0-dev/src/Column.enso [hunk 1]
```diff
@@ -411,6 +411,8 @@ type Column
              example_lt = Examples.integer_column < 1
     < : Column | Any -> Column
     < self other = Value_Type.expect_comparable self other <|
+        if other.is_a Column then
+            IO.println "AAA "+self.value_type.to_text+" :: "+other.value_type.to_text
```
Comment (radeusgd) [inline distribution/lib/Standard/Table/0.0.0-dev/src/Column.enso]: looks like leftover

File: test/Table_Tests/src/Common_Table_Operations/Column_Operations_Spec.enso [hunk 2]
```diff
@@ -649,26 +649,75 @@ add_specs suite_builder setup =
...
+            x_values = [1.25, 4.5, 5.0]
+            y_values = [2.0, 3.25, 5.0]
+
+            converters = [.truncate, x->x, dec]
+            converters.map x_converter -> converters.map y_converter ->
+                x_values_converted = x_values.map x_converter
+                y_values_converted = y_values.map y_converter
+                with_mixed_columns_if_supported [['x', x_values_converted + [Nothing]], ['y', y_values_converted + [Nothing]]
+                    clue = x.value_type.to_text+', '+y.value_type.to_text
+                    #IO.println "CLUE "+clue
```
Comment (radeusgd) [inline test/Table_Tests/src/Common_Table_Operations/Column_Operations_Spec.enso]: leftover

File: test/Table_Tests/src/Common_Table_Operations/Column_Operations_Spec.enso [hunk 3]
```diff
@@ -649,26 +649,75 @@ add_specs suite_builder setup =
...
+                            (x >= y).to_vector . should_equal (x < y).not.to_vector
+
+                            (x < 1000).to_vector . should_equal [True, True, True, Nothing]
+
+                            [(<), (<=), (>), (>=)].each op->
+                                op x y . value_type . should_equal Value_Type.Boolean
+                                op x y . to_vector . should_succeed
+                                op x 23 . to_vector . should_succeed
+                                op y 23 . to_vector . should_succeed
+                                op x 1.5 . to_vector . should_succeed
```
Comment (radeusgd) [inline test/Table_Tests/src/Common_Table_Operations/Column_Operations_Spec.enso]: Looks like this whole block is commented out. Why?

Seems like it's a duplicate with the above, isn't it? Then it probably should be removed.

File: distribution/lib/Standard/Table/0.0.0-dev/src/Column.enso (hunk 1 above)
Comment (GregoryTravis) [reply distribution/lib/Standard/Table/0.0.0-dev/src/Column.enso]: Sorry -- these changes should not have been on this branch, it was a mistake.

**Review by radeusgd — APPROVED** [review]

**Pull request merged**